garak/cli.py

```diff
@@ -174,6 +174,21 @@ def main(arguments=None) -> None:
         action="store_true",
         help="If detectors aren't specified on the command line, should we run all detectors? (default is just the primary detector, if given, else everything)",
     )
+
+    # harness
+    parser.add_argument(
+        "--harness_options",
+        type=str,
+        help="Type of harness to use. Default is probewise."
```

Comment:
If we don't hardcode a default description, we don't have to remember to update it if it changes 
```suggestion
        help="Type of harness to use."
```